tests/e2e/output_snapshots/test_search_by_non_existing_file.txt

```diff
@@ -1 +1 @@
-File "<test_images_dir>non-existing-file.jpg" not found. Check if you have typos in the filename.
+File "./non-existing-file.jpg" not found. Check if you have typos in the filename.
```

Comment:
Can you please revert this and rebase on `main`? The issue with this test was fixed on main.